File: Easy/Python/1037. Valid Boomerang.py
class Solution(object):
    def isBoomerang(self, points):
        x1 = points[0][0]
        x2 = points[1][0]
        x3 = points[2][0]
        y1 = points[0][1]
        y2 = points[1][1]
        y3 = points[2][1]

        return (x3 - x2) * (y2 - y1) != (x2 - x1) * (y3 - y2)

File: Easy/JavaScript/1037. Valid Boomerang.js
var isBoomerang = function(points) {
    let x1 = points[0][0];
    let x2 = points[1][0];
    let x3 = points[2][0];
    let y1 = points[0][1];
    let y2 = points[1][1];
    let y3 = points[2][1];

    return (x3 - x2) * (y2 - y1) != (x2 - x1) * (y3 - y2);
};
